<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Manga Reader</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="read_r2l"/>
    <addaction name="read_l2r"/>
    <addaction name="read_slide"/>
    <addaction name="separator"/>
    <addaction name="animation_key"/>
    <addaction name="no_gap"/>
    <addaction name="separator"/>
    <addaction name="copy_image"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <action name="read_r2l">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>从右向左</string>
   </property>
  </action>
  <action name="read_l2r">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>从左向右</string>
   </property>
  </action>
  <action name="animation_key">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>按键翻页动画</string>
   </property>
  </action>
  <action name="read_slide">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>上下滑动</string>
   </property>
  </action>
  <action name="no_gap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>无缝模式</string>
   </property>
  </action>
  <action name="copy_image">
   <property name="text">
    <string>复制图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
